src/i18n/de.json

```diff
@@ -893,8 +893,8 @@
     }
   },
   "pagination": {
-    "next": "Weiter",
-    "previous": "Zurück"
+    "next": "\u003E",
+    "previous": "\u003C"
```

Comment:
Is there a reason why we use entity characters instead of an Icon? 
I think it would be more esthetic if we use an icon...

Reply:
Same remark. It also doesn't really make sense to have these in the translations if it is just `<` and `>`